pages/settings/profile.vue

```diff
@@ -12,14 +12,46 @@
 		tags: selfUserInfoStore.tags?.map(tag => tag.labelName),
 	}));
 
+	const userAvatarFileInput = ref<HTMLElement>();
+	/**
+	 * 点击头像事件，模拟点击文件上传并唤起文件资源管理器
+	 */
+	function handleUploadAvatarImage() {
+		userAvatarFileInput.value?.click();
+	}
+	
+	const userUploadFile = ref<string>();
 	const avatarCropperIsOpen = ref(false);
-	const handleOpenAvatarCropper = () => {
-		avatarCropperIsOpen.value = true;
-	};
+	/**
+	 * 开启图片裁切器事件（用户选择本地文件并上传事件）
+	 * @param e 应为用户上传文件的 input 元素的 change 事件
+	 */
+	function handleOpenAvatarCropper(e: Event) {
+		const fileInput = e?.target as HTMLInputElement;
+		if (fileInput.files && fileInput.files?.[0]) {
+			const image = fileInput.files[0];
+			if (!/\.(gif|jpg|jpeg|png|bmp|GIF|JPG|PNG)$/.test(fileInput.value)) {
```

Comment:
```suggestion
			if (!/\.(jpe?g|a?png|gif|bmp|webp|avif|heic|tiff?|svg)$/i.test(fileInput.value)) {
```

可使用 `i` 标志来忽略大小写，不必将各项的大小写都罗列一遍。

同时也添加了一些常见图片格式扩展名，检查一下是不是都支持？

Reply:
![image](https://github.com/KIRAKIRA-DOUGA/KIRAKIRA-Cerasus/assets/41364445/2f275217-78ed-4004-93b8-bcb2f42bf310)

**虽然现在支持，但在 #123 后如果采用 Cloudflare Images 存储图片，bmp, avif, heic 和 tif(f) 将不受支持**

大小写问题已修复：
``` js
if (!/\.(a?png|gif|jpe?g|webp|svg)$/i.test(fileInput.value)) {
	useToast("只能上传图片文件！", "error"); // TODO 使用多语言
	console.error("ERROR", "上传的头像文件格式不合法！");
	return;
}
```